.env.sample

```diff
@@ -6,4 +6,11 @@ UNIVERSAL_ANALYTICS_GTM_CONTAINER_ID="GTM-TK92W68"
 GOOGLE_ANALYTICS_4_GTM_CONTAINER_ID="GTM-KD86CMZ"
 GA4_ENABLED="true"
 ANALYTICS_DOMAIN="localhost"
-LOG_LEVEL="debug"
\ No newline at end of file
+LOG_LEVEL="debug"
+
+# QA Env
+IPV_STUB_URL=
+F2F_FE_BASE_URL=
+TEST_HARNESS_URL=
+SESSION_TABLE=
+F2F_API_URL=
```

Comment:
Why are we not able to use the `API_BASE_URL` that already exists?

Reply:
API_BASE_URL is used for localhost testing I believe, but we can remove from .env.sample as it's set in the package.json and run-tests.sh for test runs it's required for

Reply:
The API_BASE_URL is set to a localhost URL which causes trouble with local test runs. So API_BASE_URL was introduced so we can reference the F2F BE URL for calls to the /callback endpoint

Reply:
What is the localhost url used for? Thought everything was pointing at dev envs now?

Reply:
Updated logic to remove all localhost references and use API_BASE_URL in tests